### ISSUE TITLE
CheckSignatureRule: ignoreDependencies seems inverted

### ISSUE THREAD

**Issue by famod** [mojohaus/animal-sniffer#68]
Default is `true` which (in my project) yields many many warnings like:
```
[WARNING] Skipping classes in artifact commons-discovery:commons-discovery:jar:0.4 as there are unresolved dependencies.
[WARNING] Skipping classes in artifact org.jvnet:tiger-types:jar:2.2 as there are unresolved dependencies.
[WARNING] Skipping classes in artifact org.kohsuke:asm5:jar:5.0.1 as there are unresolved dependencies.
```
The warnings vanish once I set `<ignoreDependencies>false</ignoreDependencies>` which is not what I expected after reading the documentation of this flag.
See also: https://github.com/mojohaus/animal-sniffer/blob/animal-sniffer-parent-1.18/animal-sniffer-enforcer-rule/src/main/java/org/codehaus/mojo/animal_sniffer/enforcer/CheckSignatureRule.java#L290

**Comment by famod** [comment]
Seems related: #65

**Comment by famod** [comment]
Those warnings have led my astray:
The plugin/enforcer rule _is actually trying to ignore_ the dependencies _by gathering their packages!_
This fails due to "unresolved dependencies", most likely because I added the rule to an execution with a too early phase or something.

So I don't think anymore that the implementation needs to be fixed.
